doc/release/release_0.19.rst

```diff
@@ -1,3 +1,37 @@
+scikit-image 0.19.1
+===================
+
+We're happy to announce the release of scikit-image v0.19.1!
+
+This is a small bug fix release that resolves a couple of backwards compatibility issues and a couple of issues with the wheels on PyPI. Specifically, Apple M1 wheels on PyPI were broken in 0.19.0, but should now be repaired. Wheel sizes are also greatly reduced relative to 0.19.0 by stripping debug symbols from the binaries and making sure that Cython-generated source files are not bundled in the wheels.
+
+
+Other Pull Requests
```

Comment:
Why 'Other?' Isn't this the full list of PRs making up v0.19.1 vs v0.19.0?

Reply:
It's just the standard heading for things that are uncategorised. Maybe we can rename it to "Uncategorized Pull Requests"? :shrug: I don't think it's that important...

Reply:
That header was autogenerated by the `tools/generate_release_notes.py` script. It does seem odd to list it as "other" though. It is the full list.

Reply:
Sure, no big deal, I was just curious. I think the header could write "Pull Requests included" or simply "Pull Requests..."